DTS-STN/Life-Journeys
# Modified breadcrumb component, removed pink locale bar

# Description

[LJ-262 Fix breadcrumbs or migrate to Alpha method](https://lj-decd.atlassian.net/browse/LJ-262?atlOrigin=eyJpIjoiYTM5YjVjNzNlOWRhNDM4MGEzMDNkOWIwNjk4MDA1OTQiLCJwIjoiaiJ9)

Migrated to Alpha Site's bread crumb solution but switched out the arrow they're using for the Font Awesome one we had before since that's what our designs use. I removed `BreadLink.js` and it's tests and modified the Breadcrumb tests. Also removed the pink locale bar from `layout.js`.

What this fixes: On our implementation you'll notice that when you hovered over the arrow on the Having a Baby page, it would be part of the link back to our landing page. Also with our implementation, if you're on the landing page and add a query to the end of the URL (like `/?` or `#`), breadcrumbs would appear on the page with a link back to our landing page, which was not the intended functionality. Both this issues are no longer present.

## Test Instructions

1. Pull in branch
2. Navigate to the `Having a Baby` page to see the breadcrumbs in action

## Meets Definition of Done
